packages/site/src/components/PropTable.js

```diff
@@ -150,26 +123,26 @@ function PropTableRow({
   type
 }: PropTableRowProps) {
   return (
-    <TR>
-      <PropCell>
+    <TableRow>
+      <Cell>
```

Comment:
We should try to improve the consistency of these component names, maybe `Table`, `TableRow`, `TableCell`, `TableHeaderCell` or `Table`, `Row` , `Cell`, `HeaderCell`, or similar...

Reply:
I opted for the former.